timemachine/fe/atom_mapping.py

```diff
@@ -2,10 +2,17 @@
 from typing import Optional
 
 import numpy as np
+from numpy.typing import NDArray
 from rdkit import Chem
 from rdkit.Chem import rdFMCS
 
+from timemachine.constants import DEFAULT_FF
 from timemachine.fe.topology import AtomMappingError
+from timemachine.fe.utils import set_romol_conf
+from timemachine.ff import Forcefield
+from timemachine.md.align import align_mols_by_core
+
+# def heavy_atom_mcs(a, b, threshold)
```

Comment:
remove

Reply:
Removed in https://github.com/proteneer/timemachine/pull/813/commits/c45d947dbfd2941220a5e5959ed9dd3c1b623cc3